src/main/resources/labels/tumble-dryers/tumble-dryers-2025.svg

```diff
@@ -0,0 +1,327 @@
+<svg  id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="96mm" height="192mm" viewBox="0 0 272.12601 544.25201" data-rating-increment="28.3333" data-supplier-model-chars-per-row="20">
+  <defs>
+    <style>
+      /* SI number not shown on labels until the SI is laid */
+      #siNumber {
+        display: none;
+      }
+
+      .subscale {
+        text-anchor: middle;
+        letter-spacing: 0.45px;
+      }
+
+      .subscaleA .subscaleALetter,
+      .subscaleB .subscaleBLetter,
+      .subscaleC .subscaleCLetter,
+      .subscaleD .subscaleDLetter,
+      .subscaleE .subscaleELetter {
+        font-size: 12px;
+        font-family: Verdana-Bold, Verdana;
```

Comment:
I vaguely remember that on the previous rescaled labels the letter ratings are all set in Calibri. Just checking it's correct this label uses Verdana? (I think it makes more sense for the whole label to be in Verdana anyway tbh)

Reply:
🤯 hah ok thanks for checking!